luigi_pipeline/seqr_mito_loading.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import sys
+import pkg_resources
+import pprint
+
+import luigi
+import hail as hl
+
+from lib.model.mito_mt_schema import SeqrMitoVariantsAndGenotypesSchema, SeqrMitoVariantSchema, SeqrMitoGenotypesSchema
+from luigi_pipeline.seqr_loading_optimized import BaseVCFToGenotypesMTTask, BaseMTToESOptimizedTask
+from seqr_loading import SeqrVCFToMTTask
+
+logger = logging.getLogger(__name__)
+
+
+class SeqrMitoVariantMTTask(SeqrVCFToMTTask):
```

Comment:
In seqr_loading_optimized we currently have the following:
```
class SeqrVCFToVariantMTTask(seqr_loading.SeqrVCFToMTTask):
    """
    Loads all annotations for the variants of a VCF into a Hail Table (parent class of MT is a misnomer).
    """

    def run(self):
        # We only want to use the Variant Schema.
        self.read_vcf_write_mt(schema_cls=SeqrVariantSchema)
```
In that file we should change it to 
```
class SeqrVCFToVariantMTTask(seqr_loading.SeqrVCFToMTTask):
   SCHEMA_CLASS = SeqrVariantSchema

    def run(self):
        self.read_vcf_write_mt(schema_cls=self.SCHEMA_CLASS)
```

and then this class should be
```
class SeqrMitoVariantMTTask(SeqrVCFToVariantMTTask):
   SCHEMA_CLASS = SeqrMitoVariantSchema
```

Reply:
I updated the pipeline tasks per your suggestions. Please have a look.